packages/story-editor/src/app/media/local/useContextValueProvider.js

```diff
@@ -249,6 +276,24 @@ export default function useContextValueProvider(reducerState, reducerActions) {
     ]
   );
 
+  const postProcessingBlurHashResource = useCallback(
```

Comment:
Could you elaborate on why this is not part of `postProcessingResource`?

Reply:
Because we need to one fire one at a time here. We make it part of `postProcessingResource` then it will fire all the blurhash async.